tests/pages/base.py

```diff
@@ -129,3 +133,64 @@ def click_sign_me_up(self):
         def sign_up_successful(self):
             el = self.selenium.find_element(*self._thank_you_locator)
             return el.is_displayed()
+
+    class MozillaNewsletterEmbedForm(PageRegion):
```

Comment:
As mentioned on IRC, I think we should have a base `NewsletterEmbedForm` which is then extended for additional forms. The locators, properties, and functions are mostly identical between these two page region objects.